cmd/podman/inspect/inspect.go

```diff
@@ -24,6 +26,12 @@ const (
 	ImageType = "image"
 	// ContainerType is the container type.
 	ContainerType = "container"
+	//PodType is the pod type.
+	PodType = "pod"
+	//VolumeType is the volume type
+	VolumeType = "volume"
+	//NetworkType is the network type
+	NetworkType = "network"
 	// AllType can be of type ImageType or ContainerType.
 	AllType = "all"
 )
```

Comment:
Part of this was pre-existing, but can we alphabetize the list?  It's starting to get longish to not be.